### PR TITLE
Tidied up the code for fusionChartDemo

### PR DESCRIPTION
**Pull request by bensjx**
The box and whisker chart would look weird for some activities. This is because there is only 1 data for it. If there is less than 3 data, the diagram would not look accurate. However, when you hover over it, the statistics is still accurate

### CODE REVIEW

File: src/containers/IdeaLab/Ben/fusionChartDemo.js [hunk 1]
```diff
@@ -1,274 +1,561 @@
...
-            "outlier"
-          )
-        }
-      ]
-    }
-  ]
-};
+const mapStateToProps = state => ({
+  analyticsData: state.firebase.data.analyticsData,
+  filteredAnalytics: state.fetchDataDemo.filteredAnalytics,
```
Comment (FeynmanDNA) [inline src/containers/IdeaLab/Ben/fusionChartDemo.js]: these data wont exist unless the page has visited `fetchDataDemo` route previously

File: src/containers/IdeaLab/Ben/fusionChartDemo.js [hunk 2]
```diff
@@ -1,274 +1,561 @@
...
 
-export default fusionChartDemo;
+export default compose(
+  firebaseConnect((ownProps, store) => {
+    return [
+      {
+        path: "/analytics/jupyterSolutions",
+        storeAs: "analyticsData"
+      }
+    ].concat([
```
Comment (FeynmanDNA) [inline src/containers/IdeaLab/Ben/fusionChartDemo.js]: the two objects can be inside one array without the `concat`, like `[{obj1}, {obj2}]`

File: src/containers/IdeaLab/Ben/fusionChartDemo.js (hunk 1 above)
Comment (FeynmanDNA) [reply src/containers/IdeaLab/Ben/fusionChartDemo.js]: sorry my bad, the data will be fetched due to the action called in componentDidUpdate

File: src/containers/IdeaLab/Ben/fusionChartDemo.js (hunk 2 above)
Comment (bensjx) [reply src/containers/IdeaLab/Ben/fusionChartDemo.js]: Solved in recent pull request

Comment (FeynmanDNA) [reply src/containers/IdeaLab/Ben/fusionChartDemo.js]: :+1:

**Pull request merged**